project/Component/nef/Utils/Kleisli+API.swift

```diff
@@ -5,16 +5,29 @@ import NefModels
 import Bow
 import BowEffects
 
+/// Transforms an `EnvIO<Console, nef.Error, A>` into `IO<nef.Error, A>` using a dummy console.
```

Comment:
I think this documentation is ignored.

Reply:
it is optional, but if you add it, it shows as a description, on bottom of the signature:
![Captura de pantalla 2020-03-23 a las 17 19 02](https://user-images.githubusercontent.com/25568562/77338309-7956b100-6d2a-11ea-8cf9-34528fdeb14e.png)